examples/games/game_menu.rs

```diff
@@ -149,40 +146,38 @@ mod game {
             // First create a `NodeBundle` for centering what we want to display
             .spawn((
                 NodeBundle {
-                    style: Style {
-                        // This will center the current node
-                        margin: UiRect::all(Val::Auto),
+                    // This will center the current node
+                    spacing: Spacing::AUTO_MARGIN,
+                    layout: Layout::Flex(FlexLayout {
                         // This will display its children in a column, from top to bottom
-                        flex_direction: FlexDirection::Column,
-                        // `align_items` will align children on the cross axis. Here the main axis is
-                        // vertical (column), so the cross axis is horizontal. This will center the
-                        // children
+                        direction: Direction::Column,
+                        // `align_items` will align children on the cross axis.
+                        // Here the main axis is vertical (column),
+                        // so the cross axis is horizontal.
+                        // Thus, this will center the children horizontally
```

Comment:
```suggestion
                        // As a result this will center the children horizontally
```

"Thus" is a little stuffy for docs :)